src/sentry/sentry_metrics/indexer/mock.py

```diff
@@ -11,13 +11,13 @@
     "healthy",
     "production",
     "release",
-    "session.duration",
+    "sentry.sessions.session.duration",
```

Comment:
Perhaps its better to define some global variable `sentry.sessions` and just reuse it 
Room for error is quiet high and if we ever want to replace this string with something else, would be a pita to do

Reply:
I refactored this to use the values from the enum instead. Hope that matches your intention.